Completion/Unix/Command/_stow

```diff
@@ -0,0 +1,89 @@
+#compdef stow chkstow
+
+#
+# A zsh completion script for GNU stow (https://www.gnu.org/software/stow/)
+#
+
+__stow_complete_packages() {
+  local stow_dir=${(Q)1}
+  local -a stow_pkg_list=( $stow_dir/*(-/N:t) )
+
+  if [[ ${#stow_pkg_list} -gt 0 ]]; then
+    _values -C "packages from $stow_dir" ${stow_pkg_list[@]}
+  else
+    _message "no packages found in $stow_dir"
+  fi
+}
+
+case $service in
+  stow)
+    local context state line curcontext="$curcontext" ret=1
+    typeset -A opt_args
+    # Others local variables
+    local stow_dir arguments
+
+    arguments=(
+      '(- *)'{--help,-h}'[show help]'
+      '(- *)'{--version,-V}'[show version number]'
+      '(-d --dir)'{-d+,--dir=}'[set the stow dir (default is current dir)]:stow dir:_files -/'
+      '(-t --target)'{-t+,--target=}'[set the target dir (default is parent of stow dir)]:target dir:_files -/'
+      '*'{-S,--stow}'[stow the package names that follow]:stow package:->stow_package'
+      '*'{-D,--delete}'[unstow the package names that follow]:stow package:->stow_package'
+      '*'{-R,--restow}'[restow (unstow and stow again) the package names that follow]:stow package:->stow_package'
+      '--adopt[adopt already existing plain file]'
+      '--ignore=[ignore files ending with this perl regex]:regexp:'
+      "--defer=[don't stow files beginning with this perl regex]:regexp:"
+      '--override=[force stowing files beginning with this perl regex]:regexp:'
+      '--no-folding[disable any further tree folding or tree refolding]'
+      '(-p --compat)'{-p,--compat}'[use legacy algorithm for unstowing]'
+      '(-n -no --simulate)'{-n,--no,--simulate}'[do not actually make any filesystem changes]'
+      '(-v --verbose)'{-v,--verbose}'[increase or set the level of verbosity (from 0 to 5)]'
+      '*:stow package:->stow_package'
+    )
+
+    if (( ${+commands[stow]} )) ; then
```

Comment:
Usually we don't worry about version checks like this, we just support whatever the latest options are. Especially since it's just this one option, i wouldn't bother with it personally. (If you *did* keep it, i'd try to use `_pick_variant` or something rather than making an external call every single time)

Reply:
done in 8ae64abcd